students/smitco/lesson09/mailroomOO.py

```diff
@@ -0,0 +1,147 @@
+# lesson 09 mailroom object oriented
+# !/usr/bin/env python3
+
+
+import os
+import datetime
+
+donors = {"John Travolta": [5000, 7500],
+          "Jane Fonda": [10000, 8000, 6500],
+          "Judy Blume": [3000, 3000],
+          "Joey Tribbiani": [9000],
+          "Jenny Gump": [10300, 13750, 12500]}
+
+class Donor(object):
+    def __init__(self, name, *args, **kwargs):
+       self.name = name
+    
+    def donor_name(self, name):
+        if self.name.lower() == "list":
+            print(list(donors.keys()))
+        elif self.name.lower() == "exit":
+            print("\nExiting.\n")
+        return self.name
+    
+    def donor_amount(self, name, amount):
+        self.amount = amount
+        try:
+            if self.amount.lower() == "exit":
+                print("\nExiting.")
+                self.amount = "exit"
+            elif int(self.amount) >= 10**6:
+                print("\nThe amount entered is too large.")
+                self.amount = "large"
+            else:
+                self.amount = int(self.amount)
+        except ValueError:
+            print("\nInvalid entry.")
+            self.amount = "invalid"
+        return self.amount
```

Comment:
the `donor_name` and `donor_amount` don't really belong in the Data Class. It is also unclear what function they serve? This code is probably best to be part of main program flow.